cookiecutter/hooks/post_gen_project.py

```diff
@@ -131,6 +131,27 @@ def finish_setup() -> None:
     commit_git_changes(first_commit=was_repo_initialized)
 
 
+def clean_replay_file(
+    *, replay_data: dict[str, Any], src: _pathlib.Path, dst: _pathlib.Path
+) -> dict[str, Any]:
```

Comment:
You might want to add documentation for the function?

Reply:
Oh I see it was added in a different commit

Reply:
I did, it is just in the wrong commit because I added it later and didn't saw that I put it in the wrong commit. Will fix.

Also, please don't approve if you see issues in the PR!

Reply:
Sorry, I thought it was fine to have it in the next commit and that's why I approved it

Reply:
Updated!